webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -148,79 +166,102 @@
+            >
               <span class="fa-stack">
                 <i [innerHTML]="group.types[0]?.icon | resourceIcon"></i>
                 <i class="fa fa-plus fa-stack-1x"></i>
               </span>
-              <p translate>{{group.types[0].create_key}}</p>
+              <p translate>{{ group.types[0]?.create_key }}</p>
             </a>
           </span>
 
-          <a class="mm-icon mm-icon-inverse mm-icon-caption mobile-only" href="tel:{{actionBar?.right?.sendTo?.phone}}" [ngClass]="{'mm-icon-disabled': !actionBar?.right?.sendTo?.phone}" *ngIf="actionBar?.right?.sendTo" mmAuth="can_view_call_action">
+          <a *ngIf="actionBar?.right?.sendTo"
+             mmAuth="can_view_call_action"
+             class="mm-icon mm-icon-inverse mm-icon-caption mobile-only"
+             [attr.href]="'tel:' + actionBar?.right?.sendTo?.phone | safeHtml:'url'"
+             [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
+          >
             <span class="fa fa-phone"></span>
             <p translate>call</p>
           </a>
-          <a class="mm-icon mm-icon-inverse mm-icon-caption mobile-only" href="sms:{{actionBar?.right?.sendTo?.phone}}" [ngClass]="{'mm-icon-disabled': !actionBar?.right?.sendTo?.phone}" *ngIf="actionBar?.right?.sendTo" mmAuth="can_view_message_action">
+          <a *ngIf="actionBar?.right?.sendTo"
+            mmAuth="can_view_message_action"
+            class="mm-icon mm-icon-inverse mm-icon-caption mobile-only"
+             [attr.href]="'sms:' + actionBar?.right?.sendTo?.phone | safeHtml:'url'"
+            [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
+          >
             <span class="fa fa-envelope"></span>
             <p translate>Send Message</p>
           </a>
-          <a class="mm-icon mm-icon-inverse mm-icon-caption send-message desktop-only" data-send-to="{{actionBar?.right?.sendTo?._id}}" [ngClass]="{'mm-icon-disabled': !actionBar?.right?.sendTo?.phone}" *ngIf="actionBar?.right?.sendTo" mmAuth="can_view_message_action">
+          <a *ngIf="actionBar?.right?.sendTo"
+            mmAuth="can_view_message_action"
+            class="mm-icon mm-icon-inverse mm-icon-caption send-message desktop-only"
+            [attr.data-send-to]="actionBar?.right?.sendTo?._id"
+            [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
+            (click)="actionBar?.right?.sendTo?.phone && actionBar?.right?.openSendMessageModal(actionBar?.right?.sendTo?._id)"
+          >
             <span class="fa fa-envelope"></span>
             <p translate>Send Message</p>
           </a>
 
-          <a class="mm-icon mm-icon-inverse mm-icon-caption" ui-sref="contacts.edit({ id: selectedContactDoc._id })" [ngClass]="{'mm-icon-disabled': !actionBar?.right?.canEdit}">
+          <a class="mm-icon mm-icon-inverse mm-icon-caption"
+             (click)="navigateToContactEdit(selectedContactDoc)"
```

Comment:
Please change this to be a link instead of a click event.

Reply:
Router link added